crate/static/js/main.js

```diff
@@ -39,38 +37,27 @@
 
   $('form#form-download').on('submit', function(e){
     var url = $(this).data('url');
-    var email = this.email.value;
     var uid = $.cookie('uid');
     e.preventDefault();
 
     var group = $('.form-group-email', this);
     group.removeClass('has-error');
-
-    if (email.length > 0 && uid) {
-      var valid = EMAIL_RE.test(email);
-      if (!valid) {
-        group.addClass('has-error');
-        $('.text-danger', group).removeClass('hide');
-        return;
-      }
-
+    if (uid) {
       var guid = new Date().getTime() + uid;
-      analytics.identify(uid, {
-        'email': email
-      }, {
-        'integrations': {
-          'All': false,
-          'Intercom': true
-        }
-      });
-      analytics.track('Signed up for download', {
-        'event_guid': guid,
-        'user_email': email
-      });
-    }
-    ga('send', 'event', 'button', 'click', 'download_download', 1);
-    window.open(url, '_blank');
-    this.reset();
-  });
+      analytics.identify(uid,
```

Comment:
pls also remove the segment.io tracking call, because we don't have an email any more
just keep the GA event